modules/ossm-vs-istio.adoc

```diff
@@ -115,6 +119,25 @@ spec:
 
 {SMProductName} includes CNI plug-in, which provides you with an alternate way to configure application pod networking. The CNI plug-in replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to security context constraints (SCCs) with elevated privileges.
 
+[id="ossm-global-mtls_{context}"]
+== Global mTLS settings
+{SMProductName} creates a `PeerAuthentication` resource that enables or disables Mutual TLS authentication (mTLS) within the mesh.
+
+[id="ossm-gateways_{context}"]
+== Gateways
+
+{SMProductName} installs ingress and egress gateways by default. You can disable this in the SMCP using `spec.gateways.ingress.enabled=false` or `spec.gateways.egress.enabled=false`.
+
+[id="ossm-multicluster-configuration_{context}"]
+== Multicluster configurations
+
+{SMProductName} does not provide support for multicluster configurations.
+
+[id="ossm-certificate-signing-request_{context}"]
+== Custom Certificate Signing Requests (CSR)
+
+You cannot configure {SMProductName}  to process CSRs through the Kubernetes CA.
```

Comment:
"You cannot configure {SMProductName} to process CSRs through the Kubernetes certificate authority (CA)."
Spell out the acronym for localization.

Reply:
I made the change, updated the PR, and created a new preview build.